### ISSUE TITLE
Ensure our addons are compatible with SB8

### ISSUE THREAD

**Issue by vanessayuenn** [storybookjs/storybook#26098]
Similarly to #26031, we should comb through all the addons we own to ensure they are compatible with Storybook v8.

```[tasklist]
#### Satellite repos
- [x] @storybook/test-runner
- [x] https://github.com/storybookjs/storybook/issues/26096
- [x] @storybook/addon-coverage
- [x] @storybook/addon-console
- [ ] https://github.com/storybookjs/addon-styling-webpack/pull/17
- [x] deprecate @storybook/addon-styling
- [x] @storybook/addon-svelte-csf
- [x] @storybook/addon-queryparams
- [ ] https://github.com/storybookjs/addon-designs/pull/228
- [x] @storybook/auto-config
- [ ] https://github.com/storybookjs/addon-webpack5-compiler-babel/pull/7
- [ ] https://github.com/storybookjs/addon-webpack5-compiler-swc/pull/4
```

```[tasklist]
#### Owned by Chromatic
- [x] chromatic
- [ ] https://github.com/chromaui/storybook-addon-pseudo-states/pull/100
- [x] @chromaui/addon-visual-tests
```

### In monorepo 
_Should already be compatible but should double check_
- [ ] ~@storybook/postinstall~
- [x] @storybook/addon-storysource
- [x] @storybook/addon-jest
- [x] @storybook/addon-themes
- [x] @storybook/addon-mdx-gfm
- [x] @storybook/addon-actions
- [x] @storybook/addon-docs
- [x] @storybook/addon-viewport
- [x] @storybook/addon-controls
- [x] @storybook/addon-backgrounds
- [x] @storybook/addon-toolbars
- [x] @storybook/addon-essentials
- [x] @storybook/addon-measure
- [x] @storybook/addon-outline
- [x] @storybook/addon-links
- [x] @storybook/addon-highlight
- [x] @storybook/addon-interactions
- [x] @storybook/addon-a11y

**Comment by JReinhold** [comment]
The `addon-styling-webpack` 1.0 release PR should be merged when 8.0 goes GA. I'll take care of that.

**Comment by JReinhold** [comment]
`@storybook/addon-events` is not maintained anymore.
